docker/run_crontabber.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
is this a reasonable entrypoint/cmd for a production crontabber container?
I think it should probably be OK, but I'm not sure.